motor_control/throttle/Button.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BUTTON_THROTTLE
+#define BUTTON_THROTTLE
+
+#include "base_throttle.h"
+
+class Button: public Throttle{
+    public:
+    Button(int ft);
+    virtual int readThrottle();
+    int mappingFunction(int f, int s);
```

Comment:
Helper functions are awesome, so I like that this is included, but it should be a private member function because the user should not need/have access to this. Also, we need better variable names than `f` and `s`.